_game/teams/Plague

```diff
@@ -5,3 +5,9 @@ Plague Team wins when all players are either dead, unaligned or part of the Plag
 __Formalized__
 Win Condition: @(Alignment:Plague), @(Class:Unaligned)
 On Join: Add @Joiner to #Plague-Pit
+Passive Start Phase:
+  • Process: Investigate @(Attr:Contaminated) Player Count
+  • Evaluate: 
+    ‣ @Result is 0: Reveal `No one has the Plague` to #plague-pit
+    ‣ @Result is 1: Reveal `@Result->Number Player has the Plague` to #plague-pit
```

Comment:
Should we use "player" here instead of "Player"? Note that @Result->Number turns into "One", "Two", etc and not "1", "2"